packages/codebytes/src/index.ts

```diff
@@ -1,2 +1,2 @@
 export * from './codeByteEditor';
-export { LanguageOption, LanguageOptions } from './consts';
+export * from './consts';
```

Comment:
this was blocking us from being able to import in the monolith for some reason, so we're just gonna do this to unblock